src/cli.ts

```diff
@@ -1,44 +1,160 @@
 #!/usr/bin/env node
+import {syncPackages} from "./index"
+import * as inquirer from "inquirer"
+import {isValidUrl} from "./utils"
+import {MigrationMode, MigrationSettings} from "./settings"
 
-// Parse command line
-// @ts-ignore(TS1208): all files must be modules when the '--isolatedModules' flag is provided
-const {name, from, to, dryRun} = require('yargs')
-  .usage(`Usage: $0 --name <name> --from.registry <url> [--from.token <x>] --to.registry <url> [--to.token <y>] [--dry-run]\n
-Publish package versions from one registry to another.`)
-  .example('$0 --name @scope/name --from.registry https://registry.npmjs.org/ --from.token $NPM_TOKEN ' +
-    '--to.registry https://npm.pkg.github.com --to.token $GITHUB_TOKEN')
+const logger = require('pino')()
+
+const {packages, from, to, dryRun, latestOnly, latestMajors, repository} = require('yargs')
+  .usage("Usage: $0 --packages <packages> --from.registry <url> [--from.token <x>] --to.registry <url> [--to.token <y>] " +
+           "[--dry-run] [--latest-only] [--latest-majors] [--repository https://github.com/joebowbeer/regsync]\n" +
+           "Publish package versions from one registry to another.")
+  .example('$0 --packages @scope/packageName1 @scope/packageName2 --from.registry https://registry.npmjs.org/ --from.token $NPM_TOKEN ' +
+             '--to.registry https://npm.pkg.github.com --to.token $GITHUB_TOKEN',
```

Comment:
Codacy has a fix for the issue: [Replace `·····'--to.registry·https://npm.pkg.github.com·--to.token·$GITHUB_TOKEN'` with `"--to.registry·https://npm.pkg.github.com·--to.token·$GITHUB_TOKEN"`](https://app.codacy.com/gh/joebowbeer/regsync/pullRequest?prid=9263162)
```suggestion
        "--to.registry https://npm.pkg.github.com --to.token $GITHUB_TOKEN",
```